src/api_routes_http.rs

```diff
@@ -291,7 +292,14 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimitCell) {
           .route("/verify_email", web::post().to(verify_email))
           .route("/leave_admin", web::post().to(leave_admin))
           .route("/totp/generate", web::post().to(generate_totp_secret))
-          .route("/totp/update", web::post().to(update_totp)),
+          .route("/totp/update", web::post().to(update_totp))
+          .route("/export", web::get().to(export_user_backup)),
```

Comment:
Should there be a stricter rate limit for export too?

Reply:
Ya pry a good idea, the strictest we have.